src/support/azext_support/commands.py

```diff
@@ -3,59 +3,108 @@
-                            client_factory=cf_communications) as g:
-        g.custom_command('list', 'list_support_tickets_communications')
-        g.custom_show_command('show', 'get_support_tickets_communications')
-        g.custom_command('create', 'create_support_tickets_communications', supports_no_wait=False)
-     
-    self.command_table['support in-subscription tickets update'] = TicketUpdate(loader=self)
-    self.command_table['support in-subscription tickets create'] = TicketCreate(loader=self)
-    self.command_table['support in-subscription communication create'] = CommunicationCreate(loader=self)
-    self.command_table['support no-subscription communication create'] = CommunicationNoSubscriptionCreate(loader=self)
+        operations_tmpl="azext_support.vendored_sdks.operations#CommunicationsOperations.{}",
+        client_factory=cf_communications,
+    )
+
+    self.command_group("support", support, client_factory=cf_support)
+
+    with self.command_group(
+        "support services", support_services, client_factory=cf_services
+    ) as g:
+        g.command("list", method_name="list")
+        g.show_command("show", getter_name="get")
+
+    with self.command_group(
+        "support services problem-classifications",
+        support_problem_classifications,
+        client_factory=cf_problem_classifications,
+    ) as g:
+        g.command("list", method_name="list")
+        g.show_command("show", getter_name="get")
+
+    with self.command_group(
+        "support tickets", support_tickets, client_factory=cf_support_tickets
+    ) as g:
+        g.custom_command("list", "list_support_tickets")
+        g.custom_show_command("show", "get_support_tickets")
+        g.custom_command(
+            "create",
+            "create_support_tickets",
+            supports_no_wait=False,
+            validator=validate_tickets_create,
+        )
+        g.custom_command("update", "update_support_tickets")
+
+    with self.command_group(
+        "support tickets communications",
+        support_communications,
+        client_factory=cf_communications,
+    ) as g:
+        g.custom_command("list", "list_support_tickets_communications")
+        g.custom_show_command("show", "get_support_tickets_communications")
+        g.custom_command(
+            "create", "create_support_tickets_communications", supports_no_wait=False
+        )
+
+    self.command_table["support in-subscription tickets update"] = TicketUpdate(
+        loader=self
+    )
+    self.command_table["support in-subscription tickets create"] = TicketCreate(
+        loader=self
+    )
+    self.command_table[
+        "support in-subscription communication create"
+    ] = CommunicationCreate(loader=self)
+    self.command_table[
+        "support no-subscription communication create"
+    ] = CommunicationNoSubscriptionCreate(loader=self)
+
+    with self.command_group("support no-subscription file") as g:
```

Comment:
Doesnt this also need "upload" in its command. The command you have here could mean any operation under file